src/Workspace/ErrorWorkspacePlaceholder.jsx

```diff
@@ -1,21 +1,33 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom/cjs/react-router-dom';
+import isEnabled from '../helpers/featureFlags';
 import './Workspace.less';
 import NotFoundSVG from '../img/not-found.svg';
 import { components } from '../params';
+import workspaceRegistrationConfig from '../localconf';
 
-class ErrorWorkspacePlaceholder extends React.Component {
-  render() {
-    const supportEmail = components.login?.email || 'support@datacommons.io';
-    return (
-      <div className='error-workspace-placeholder__error-msg'>
-        <h1>Error opening workspace...</h1>
-        <p>
-        Workspace access requires authorization. Please contact <a href={`mailto:${supportEmail}`}>{supportEmail}</a> for more information.
-        </p>
-        <NotFoundSVG />
-      </div>
-    );
+const ErrorWorkspacePlaceholder = () => {
+  const history = useHistory();
+  if (isEnabled('workspaceRegistration')) {
+    if (workspaceRegistrationConfig) {
+      // eslint-disable-next-line react/prop-types
+      history.push('/workspace/register');
+    } else {
+      // eslint-disable-next-line no-console
+      console.log('Unable to display registration form. Missing workspaceRegistrationConfig');
```

Comment:
I think this information doesn't needs a print out to the client side

Reply:
Oh! Is it? Thanks for letting me know. I can take it off, but I was wondering what would be the best way to let the developer know if this is what they're missing if a situation like this arises?

Reply:
We usually just put these kind of print out in adhocly when doing development, and remove them before shipping to production